### PR TITLE
NuScenes plugin

### PR DESCRIPTION
**Pull request by DuncanZauss**
- NuScenes Data plugin
- Evaluation with MS COCO style metrics
- Torchhub downloadable pretrained shufflenetv2k16 model for NuScenes object detection

### CODE REVIEW

**Comment by svenkreiss** [comment]
That's great @DuncanZauss .

Maybe put it on your todo list to document this with a tutorial or at least to extend the class docstring with some information on how to download data and how to unpack and copy it to what destination. I might add something so that all data loader docstrings get rendered into the Guide same as PyTorch documents all classes. Then the documentation lives closer to the code. Happy to hear your thoughts on the best way to document, too.

**Comment by DuncanZauss** [comment]
Thanks for accepting the PR @svenkreiss .
I will add a section to the Guide soon and I can certainly also add a quick-start guide as the class docstring, so that it can get rendered alongside the other data module docstrings. I agree, that it's nice if the documentation is closer to the code, but the Guide is also great. Especially, since we can show qualitative results as "advertisement", which I think is a good visual way of convincing people to use OpenPifPaf for their applications.
